src/GooglePlay/AbstractResponse.php

```diff
@@ -36,6 +36,14 @@ public function getDeveloperPayload()
         return $this->response->getDeveloperPayload();
     }
 
+    /**
+     * @return bool
+     */
+    public function isAcknowledge()
```

Comment:
this method should looks like that
```
    /**
     * @return bool
     */
    public function isAcknowledged()
    {
        return $this->response->acknowledgementState === static::ACKNOWLEDGEMENT_STATE_DONE;
    }
```

Reply:
Do not forget add the described in docs values:
```
    const ACKNOWLEDGEMENT_STATE_YET_TO_BE = 0;
    const ACKNOWLEDGEMENT_STATE_DONE = 1;
```
smth like that

Reply:
Because of Google migth add in the future another values for acknowledment state
we need a getter
```
    /**
     * @return integer
     */
    public function getAcknowledgementState()
    {
        return $this->response->acknowledgementState;
    }
```